Client.xcodeproj/project.pbxproj

```diff
@@ -17931,6 +17931,7 @@
 				PROVISIONING_PROFILE_SPECIFIER = "bitrise org.mozilla.ios.Firefox";
 				SKIP_INSTALL = NO;
 				SWIFT_OBJC_BRIDGING_HEADER = "$(PROJECT_DIR)/Client/Client-Bridging-Header.h";
+				SWIFT_VERSION = 5.0;
```

Comment:
`SWIFT_VERSION = 5.0;` change necessary, was done automatically? 🤔 Wondering why

Reply:
Taking a look because the change was not intentional

Reply:
Fixed